### PR TITLE
Add note on how to handle Prometheus replicas.

### PR DESCRIPTION
**Pull request by frankreno**
###### Description

Update docs with a note on support and handling for multiple Prometheus replicas.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/existingPrometheusDoc.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # How to install if you have an existing Prometheus operator
 
+__NOTE__: The Sumo Logic Kubernetes collection process does not support collecting metrics from when scaling Prometheus replicas. If you are running multiple Prometheus replicas, please follow our [Side-by-Side](SideBySidePrometheus.md) instructions.
```
Comment (MariePMcGarry) [inline deploy/docs/existingPrometheusDoc.md:3]: I suspect missing or extra word here: "metrics from when"

File: deploy/docs/standAlonePrometheus.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # How to install if you have standalone Prometheus
 
+__NOTE__: The Sumo Logic Kubernetes collection process does not support collecting metrics from when scaling Prometheus replicas. If you are running multiple Prometheus replicas, please follow our [Side-by-Side](SideBySidePrometheus.md) instructions.
```
Comment (MariePMcGarry) [inline deploy/docs/standAlonePrometheus.md:3]: I suspect missing or extra word here: "metrics from when"

File: deploy/docs/existingPrometheusDoc.md (hunk 1 above)
Comment (frankreno) [reply deploy/docs/existingPrometheusDoc.md]: good catch, fixed.

File: deploy/docs/standAlonePrometheus.md (hunk 2 above)
Comment (frankreno) [reply deploy/docs/standAlonePrometheus.md]: good catch, fixed.

**Pull request merged**